app/Jobs/NotifyUserOfNewGistComments.php

```diff
@@ -69,6 +75,13 @@ public function handle(GistClient $gistClient, GitHubClient $githubClient)
         }
     }
 
+    private function gistCreatedAfterTheDayOfReckoning($gist)
+    {
+        $date = Carbon::createFromFormat('Y-m-d\TH:i:s\Z', $gist['created_at']);
+        // Compare against the day GitHub added gist comment notifications *for new gists*
+        return $date->greaterThan(Carbon::createFromDate(2019, 5, 8));
```

Comment:
This needs to specify UTC and the start of the day. Otherwise, it'll send Giscus notifications for gists created today (tested this locally).

```suggestion
        return $date->greaterThan(Carbon::createFromDate(2019, 5, 8, 'UTC')->startOfDay());
```